docs/guide/contribute/contribute-code.md

```diff
@@ -79,6 +79,14 @@ If you are writing Java or C++ with Android Studio, **License** will be added to
     * If the PR is about adding a new feature, another [PR for documentation](how-to-contribute.html#contribute-code-or-document) is necessary in codes' PR description.
     * *Optional* If the PR fixes an existing Github issue, you may add the link to the corresponding issue in the PR.
 
+There will be a static check program when you submit a PR,and the following rules will be checked:
+1. Check if your PR is submitted to master branch, if not, you will failed.
+2. Check if your PR is binded with a milestone, if not, you will receive a warning message.
+3. Check if your PR description contains keywords `Document` and it's correponding http links. if not, you will receive a warning message.

```

Comment:
`Document` is probably not what you mean here, but `Documentation`

Reply:
in the PR description template, it use `Document:http-link`  to describe the correponding weex document link of PR.
so the rule is check if the description contains   `Document:http-link`.

so I use Document instead of Documentation here

Reply:
Well, I think you should change both [PR template](https://github.com/apache/incubator-weex/pull/2676/) and document here to `Documentation`

Reply:
ok, I will change it.